src/taglibs/migrate/util/addIdScopedAttr.js

```diff
@@ -0,0 +1,16 @@
+module.exports = function addIdScopedAttr(context, el, attrValue) {
+    const templateRoot = el.parentNode;
```

Comment:
`el.parentNode` is not the actual template root. We actually want to use `context.root`.